website/docs/advanced/tenancy.md

```diff
@@ -11,37 +11,63 @@ This feature is in alpha. Use at your own risk.
 ::::
 
 ::::info
-This feature is available from 2024.2 and is not to be confused with brands, which used to be called tenants.
+This feature is available from 2024.2 and is not to be confused with [brands](../core/brands.md), which were previously called tenants.
 ::::
 
-## Preparations
+## About tenants
 
-Starting with 2024.2, authentik allows an administrator or operator to create multiple tenants. This means that an operator can manage several authentik installations without having to deploy additional instances.
+Starting with version 2024.2, authentik allows an administrator or operator to create multiple tenants. This means that an operator can manage several different and distinct authentik installations, each with it's own Install ID and license(s). The relationships between tenant and installation (Install ID) is a 1:1 relationship, so for each tenant, there is a unique Install ID.
```

Comment:
I feel like under this would be a good place to move the warning about expression policies not being separated yet.

Reply:
Sure, I currently have it under the "Important Considerations" section, but it could move up. The way it is now is sort of the same "progressive disclosure".. first show the basics, then show the "You really need to know this thing", and then fianaly the How To steps.

But yeah we can move up if we want them to absolutely see it first, because it is what might break everything.

Reply:
Yeah that was my thinking. I also wonder if we can make it red instead of yellow.